readers/mongodb/setup_test.go

```diff
@@ -23,13 +25,18 @@ func TestMain(m *testing.M) {
 		testLog.Error(fmt.Sprintf("Could not connect to docker: %s", err))
 	}
 
-	cfg := []string{
-		"MONGO_INITDB_DATABASE=test",
-	}
-
-	container, err := pool.Run("mongo", "4.4.3-bionic", cfg)
+	container, err := pool.RunWithOptions(&dockertest.RunOptions{
+		Repository: "mongo",
+		Tag:        "6.0.13",
```

Comment:
There is a newer version
```suggestion
		Tag:        "7.0.5",
```